README.md

```diff
@@ -526,15 +526,44 @@ enum ImageOrientation {
 
 | Param | Type |
 | --- | --- |
-| environment | Environment | 
-| [venueApiUrl] | string | 
+| environment | Environment |
+| [venueApiUrl] | string |
 
 * getSeatAttributes(venueId) ⇒ Array&lt;SeatAttributes&gt;
+* getSeatAttributesBySeatId(venueId, seatIdCollection) ⇒ Array&lt;SeatAttributes&gt;
+* getDetails(venueId, seatIdCollection) ⇒ Array&lt;VenueDetails&gt;
```

Comment:
Please, correct the documentation since it's different from the actual interfaces

Reply:
done